pkg/cli/cmd/certs.go

```diff
@@ -0,0 +1,162 @@
+package cmd
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aserto-dev/topaz/pkg/cli/cc"
+	"github.com/aserto-dev/topaz/pkg/cli/certs"
+)
+
+type CertsCmd struct {
+	List     ListCertsCmd     `cmd:"" help:"list dev certs"`
+	Generate GenerateCertsCmd `cmd:"" help:"generate dev certs"`
+	Trust    TrustCertsCmd    `cmd:"" help:"trust dev certs"`
+}
+
+const (
+	DefaultCertsDir = "~/.config/topaz/certs"
+
+	gatewayCertFileName = "gateway"
+	grpcCertFileName    = "grpc"
+	certCommonName      = "topaz"
+)
+
+type ListCertsCmd struct {
+	CertsDir string `arg:"" required:"false" default:"~/.config/topaz/certs" help:"path to dev certs folder" `
+}
+
+func (cmd ListCertsCmd) Run(c *cc.CommonCtx) error {
+	certsDir := cmd.CertsDir
+	if cmd.CertsDir == DefaultCertsDir {
+		home, err := os.UserHomeDir()
+		if err != nil {
+			return err
+		}
+
+		certsDir = path.Join(home, "/.config/topaz/certs")
+	}
+
+	files, err := os.ReadDir(certsDir)
+	if err != nil {
+		return err
+	}
+	certDetails := make(map[string]*x509.Certificate)
+	for _, file := range files {
+		if file.IsDir() || !strings.HasSuffix(file.Name(), ".crt") {
+			continue
+		}
+
+		content, err := os.ReadFile(filepath.Join(certsDir, file.Name()))
+		if err != nil {
+			return err
+		}
+		block, _ := pem.Decode(content)
+		cert, err := x509.ParseCertificate(block.Bytes)
+		if err != nil {
+			return err
+		}
+		certDetails[file.Name()] = cert
+	}
+	table := c.UI.Normal().WithTable("File", "Not Before", "Not After", "Valid")
+
+	fileNames := make([]string, 0, len(certDetails))
+	for k := range certDetails {
+		fileNames = append(fileNames, k)
+	}
+
+	sort.Strings(fileNames)
+
+	table.WithTableNoAutoWrapText()
+	for _, k := range fileNames {
+		isValid := true
+		if time.Until(certDetails[k].NotAfter) < 0 {
+			isValid = false
+		}
+		table.WithTableRow(k, certDetails[k].NotBefore.Format(time.RFC3339), certDetails[k].NotAfter.Format(time.RFC3339), fmt.Sprintf("%t", isValid))
+	}
+	table.Do()
+	return nil
+}
+
+type GenerateCertsCmd struct {
```

Comment:
We provide `list`, `generate` and `trust` 

How does one :
* remove certs
* remove trust
* regenerate certs when certs are expired